runner/dl_checkpoints.sh

```diff
@@ -164,6 +164,18 @@ function build_tensorrt_models() {
       echo "failed ComfyUI build_trt.py dynamic engine"
       return 1
     )
+
+  # FasterLivePortrait
+  docker run --rm -v ./models:/models --gpus all -l TensorRT-engines $AI_RUNNER_COMFYUI_IMAGE \
+    bash -c "/workspace/ComfyUI/custom_nodes/ComfyUI-FasterLivePortrait/scripts/build_fasterliveportrait_trt.sh \
+             /workspace/ComfyUI/models/liveportrait_onnx && \
```

Comment:
This script requires 3 parameters https://github.com/pschroedl/ComfyUI-FasterLivePortrait/blob/bc579dbb14b9bed6a55f39dcdbdfa5d2ab66cff8/scripts/build_fasterliveportrait_trt.sh#L10-L12, I had to modify paths for it to run